lucene/tools/src/groovy/install-markdown-filter.groovy

```diff
@@ -28,7 +28,7 @@ import com.vladsch.flexmark.html.HtmlRenderer;
 import com.vladsch.flexmark.parser.Parser;
 import com.vladsch.flexmark.parser.ParserEmulationProfile;
 import com.vladsch.flexmark.util.html.Escaping;
-import com.vladsch.flexmark.util.options.MutableDataSet;
+import com.vladsch.flexmark.util.data.MutableDataSet;
```

Comment:
Is this intentional? looks like the precommit fails because of this

Reply:
Reverted.